### PR TITLE
FA-5: Create the add poll action sub-app.

### PR DESCRIPTION
**Pull request by nanoeagle**

### CODE REVIEW

File: voting-contentApp/i18n/voting-app-language_en.properties [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#For App
+voting-app.detail.label: Poll Data
+
+#For Detail SubApp - The Form
+voting-app.jcrName.label: Title
```
Comment (ThuyTo) [inline voting-contentApp/i18n/voting-app-language_en.properties]: Maybe change voting-app.jcrName.label: Title to voting-app.jcrName.label=Title. i18n key should use ' =', not ':'. Please change all keys.

File: voting-contentApp/apps/voting-app.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+!content-type:poll
```
Comment (llamkhan) [inline voting-contentApp/apps/voting-app.yaml]: This is an old definition style, should follow with latest documentation. BTW, this one should be go after FA-2 vs 4

Comment (nanoeagle) [reply voting-contentApp/apps/voting-app.yaml]: I have rechecked the latest document, but it tells the same thing as I defined. Document reference here:
https://docs.magnolia-cms.com/product-docs/6.2/Apps/My-first-content-app/Part-I---Creating-a-basic-content-app.html#_customizing_the_app_descriptor

File: voting-contentApp/i18n/voting-app-language_en.properties (hunk 1 above)
Comment (nanoeagle) [reply voting-contentApp/i18n/voting-app-language_en.properties]: Yes. I followed the old files on document. I am changing these characters from ":" to "=".

**Review by llamkhan — APPROVED** [review]

File: voting-contentApp/i18n/voting-app-language_en.properties [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+voting-app.detail.label = Poll Data
+voting-app.name.label = Title
+voting-app.description.label = Description
+voting-app.imageLink.label = Select Image
+voting-app.expiryDate.label = Expiry Date
+voting-app.question.label = Add Question
+voting-app.option1.label = Required Option 1
+voting-app.option2.label = Required Option 2
+voting-app.optionX.label = Extra Options
+voting-app.optionX.value.label =
```
Comment (llamkhan) [inline voting-contentApp/i18n/voting-app-language_en.properties]: is this redundant property?

**Review by ThuyTo — APPROVED** [review]

File: voting-contentApp/i18n/voting-app-language_en.properties (hunk 3 above)
Comment (nanoeagle) [reply voting-contentApp/i18n/voting-app-language_en.properties]: No, it is not. The last statement with the empty value is for the porpuse of hiding default "voting-app.optionX.value.label" lable of each extra obtion.

**Pull request merged**